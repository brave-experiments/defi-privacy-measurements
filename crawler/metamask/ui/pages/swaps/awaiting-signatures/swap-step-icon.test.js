import React from 'react';

import { renderWithProvider } from '../../../../test/jest';
import SwapStepIcon from './swap-step-icon';

describe('SwapStepIcon', () => {
  it('renders the component', () => {
    const { container } = renderWithProvider(<SwapStepIcon />);
    expect(container).toMatchSnapshot();
  });
});
