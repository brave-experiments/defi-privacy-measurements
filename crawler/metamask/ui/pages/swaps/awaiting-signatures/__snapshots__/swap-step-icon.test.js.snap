// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwapStepIcon renders the component 1`] = `
<div>
  <svg
    fill="none"
    height="14"
    viewBox="0 0 14 14"
    width="14"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="7"
      cy="7"
      r="6.25"
      stroke="#037DD6"
      stroke-width="1.5"
    />
    <path
      d="M6.50983 5.192H5.27783L6.14183 4H7.71783V9.68H6.50983V5.192Z"
      fill="#037DD6"
    />
  </svg>
</div>
`;
